packages/xo-server/src/api/remote.js

```diff
@@ -15,6 +15,16 @@ get.params = {
   id: { type: 'string' },
 }
 
+export async function getInfo({ id }) {
+  return this.getRemoteInfo(id)
+}
```

Comment:
Is it used somewhere?

Reply:
I had it for xo-cli, but i can remove it